tasks/install-jbig2enc.yml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Yeah, have not got round to that bit yet. That is in the plans for the next PR. I spent quite a chunk of time on the role this week....it will even build on aarch64 (pros/cons of having an m1 macbook)

Reply:
Yeah, I'm not sure how best to handle it, since I doubt it appears in something like `package_facts` output.  But it's alright for now, it's not that intensive to build at least.